### PR TITLE
build: add lint rule "prefer-const"

### PR DESCRIPTION
**Pull request by sandrozbinden**

### CODE REVIEW

File: e2e/entries.e2e-spec.ts [hunk 1]
```diff
@@ -9,9 +9,9 @@ describe('Entries E2E Test', () => {
 
   let originalTimeout;
   let loginPage = new LoginPageObject();
```
Comment (mleu) [inline e2e/entries.e2e-spec.ts]: linter thinks this should be a `const`

File: e2e/settings.e2e-spec.ts [hunk 2]
```diff
@@ -8,10 +8,10 @@ import { UploadPageObject } from './page-objects/upload-page-object';
 describe('Settings E2E Test', () => {
 
   let originalTimeout;
-  let settingsPage = new SettingsPageObject();
+  const settingsPage = new SettingsPageObject();
   let loginPage = new LoginPageObject();
```
Comment (mleu) [inline e2e/settings.e2e-spec.ts]: linter thinks this also should be a `const`

Comment (sandrozbinden) [inline e2e/settings.e2e-spec.ts]: Thanks correct i "fixed" those in the review  and realized they have been initalizied twice

File: e2e/entries.e2e-spec.ts (hunk 1 above)
Comment (sandrozbinden) [inline e2e/entries.e2e-spec.ts]: Thanks correct i "fixed" those in the review  and realized they have been initalizied twice

**Pull request merged**